renderer.go

```diff
@@ -222,12 +222,7 @@ func (r *Renderer) renderThematicBreak(node ast.Node, entering bool) ast.WalkSta
 	if entering {
 		breakChars := []byte{'-', '*', '_'}
 		breakChar := breakChars[r.config.ThematicBreakStyle : r.config.ThematicBreakStyle+1]
-		var breakLen int
-		if r.config.ThematicBreakLength < ThematicBreakLengthMinimum {
-			breakLen = int(ThematicBreakLengthMinimum)
-		} else {
-			breakLen = int(r.config.ThematicBreakLength)
-		}
+		breakLen := int(max(r.config.ThematicBreakLength, ThematicBreakLengthMinimum))
```

Comment:
`max` is available since Go 1.21, that is the minimum version set in `go.mod`, so it should be safe to be used here.